packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -437,11 +437,12 @@ const VirtualListBaseFactory = (type) => {
 				const nextRow = (nextIndex - nextColumn) % dataSize / dimensionToExtent;
 				const numOfItemsInPage = Math.floor((clientSize + spacing) / gridSize) * dimensionToExtent;
 				const firstFullyVisibleIndex = Math.ceil(scrollPosition / gridSize) * dimensionToExtent;
+				const isCurrentItemInView = index >= firstFullyVisibleIndex && index < firstFullyVisibleIndex + numOfItemsInPage;
```

Comment:
It just finally clicked with me why I don't like these variable names...  They imply a question, so they make better _function_ names than _variable_ names.  `isCurrentItemInView` asks the question, `CurentItemIsInView` holds the answer.  I'm not suggesting a change, just making the observation.